tracker.py

```diff
@@ -8,7 +8,7 @@
 
 
 class Tracker:
-    def __init__(self, path):
+    def __init__(self, path, torrent):
```

Comment:
Needed to connect `Tracker`, `PeerConnection`, and `DownloadHandler` together to show time speeds.